.github/workflows/cluster_endtoend_12.yml

```diff
@@ -9,12 +9,15 @@ concurrency:
 env:
   LAUNCHABLE_ORGANIZATION: "vitess"
   LAUNCHABLE_WORKSPACE: "vitess-app"
-  GITHUB_PR_HEAD_SHA: "${{ github.event.pull_request.head.sha }}"
+  EXPERIMENTAL_GITHUB_OIDC_TOKEN_AUTH: 1
 
 jobs:
   build:
     name: Run endtoend tests on Cluster (12)
     runs-on: ubuntu-20.04
+    permissions:
+      id-token: write
+      contents: read
```

Comment:
Thank you for fixing the `permissions` key, it seems to run fine now 🥳

However, can we limit the scope to `id-token: write` only? Your documentation only asks for `id-token`, not `contents`.

Reply:
Please see the commit message. tldr is that contents: read is given by default if unspecified, and this is added to partially recreate the default. The existing steps implicitly have been using it. Launchable's CLI itself won't need it.

Reply:
https://docs.github.com/en/actions/security-guides/automatic-token-authentication#permissions-for-the-github_token is the default config when "permissions" part doesn't exist. If we drop "contents: read", the e2e will fail at `actions/checkout@v3`.

Reply:
Awesome! Looks good to me :)